### PR TITLE
Implement cycling through timezones in "time" block on click

### LINKED ISSUE #1658: Cycle through timezones in "time" block on click
Unless I didn't read through the docs close enough, this currently isn't possible

### PR DESCRIPTION
**Pull request by omkar-mohanty**
Fixes #1658

### CODE REVIEW

File: src/blocks/time.rs [hunk 1]
```diff
@@ -38,17 +38,29 @@ pub struct Config {
     format: FormatConfig,
     #[default(1.into())]
     interval: Seconds,
-    timezone: Option<Tz>,
+    timezones: Option<Vec<Tz>>,
```
Comment (MaxVerevkin) [inline src/blocks/time.rs:41]: This is a breaking change but it doesn't need to be. See #1452 for very similar example. TL;DR: You can create an enum `Timezone` which is either `Tz` or `Vec<Tz>`. `#[serde(untagged)]` will handle the deserialization.

File: src/blocks/time.rs [hunk 2]
```diff
@@ -38,17 +38,29 @@ pub struct Config {
...
 }
 
 pub async fn run(config: Config, mut api: CommonApi) -> Result<()> {
+    // "toggle" changes the current displayed timezone to the timezone next in the list.
+    // for example in the list ["Asia/Kolkata", "Europe/London"] if the current displayed timezone is
+    // "Europe/London" on "toggle" the timezone changes to "Asia/Kolkata"
+    api.set_default_actions(&[(MouseButton::Left, None, "toggle")]).await?;
```
Comment (MaxVerevkin) [inline src/blocks/time.rs:48]: Perhaps `next_timezone` would be a better name? New actions can be added later and "toggle" is a bit too generic.

Also please document this action. For example, see [net docs](https://greshake.github.io/i3status-rust/i3status_rs/blocks/net/index.html).

Comment (omkar-mohanty) [reply src/blocks/time.rs]: Thanks for the review! I added the documentation for `next_timezone` action.

**Comment by omkar-mohanty** [comment]
@MaxVerevkin I think i have implemented all the suggestions can I get a review? Thanks!

**Review by MaxVerevkin — APPROVED** [review]

**Pull request merged**